api/grpc/metrics/api.proto

```diff
@@ -29,13 +29,23 @@ message AuditRequest {
         SECCOMP = 0;
         SELINUX = 1;
     }
+    message SeccompAuditReq {
+        string syscall = 1;
+    }
+    message SelinuxAuditReq {
+        string perm = 1;
+        string scontext = 2;
+        string tcontext = 3;
+        string tclass = 4;
+    }
     Type type = 1;
```

Comment:
In theory we could remove the type in favor of checking those against `nil`:
```
SeccompAuditReq seccompReq = 7;
SelinuxAuditReq selinuxReq = 8;
```

Reply:
yes, good idea